backend/src/main/scala/cromwell/backend/BackendActorFactory.scala

```diff
@@ -0,0 +1,19 @@
+package cromwell.backend
+
+import akka.actor.{ActorRef, ActorSystem}
+import cromwell.backend.model.WorkflowDescriptor
+
+trait BackendActorFactory {
+
+  /**
+    * Returns a backend actor instance based on the init class, Cromwell actor system and specific workflow.
+    *
+    * @param initClass          Initialization class for the specific backend.
+    * @param actorSystem        Cromwell Engine actor system.
```

Comment:
Technically this would be the actor system designated for backend actors, not engine actors :)